litmus-portal/graphql-server/graph/schema.resolvers.go

```diff
@@ -486,3 +481,14 @@ func (r *Resolver) Subscription() generated.SubscriptionResolver { return &subsc
 type mutationResolver struct{ *Resolver }
 type queryResolver struct{ *Resolver }
 type subscriptionResolver struct{ *Resolver }
+
+// !!! WARNING !!!
+// The code below was going to be deleted when updating resolvers. It has been copied here so you have
+// one last chance to move it out of harms way if you want. There are two reasons this happens:
+//  - When renaming or deleting a resolver the old code will be put in here. You can safely delete
+//    it when you're done.
+//  - You have helper methods in this file. Move them out to keep these resolver files clean.
+const (
+	AcceptedInvitation = "Accepted"
+	PendingInvitation  = "Pending"
+)
```

Comment:
@rajdas98 can we pass it into the utils for now or the types/config file?

Reply:
Done, have put in resolver.go

Reply:
If it has to be removed should be in usermanagement pkg, not in resolvers